scripts/get_counts.py

```diff
@@ -353,8 +359,8 @@ def GetCountsMultiProcess(source_int_dir, dest_count_dir, ngram_order, n, num_pr
                'trap "rm -r {0}" SIGINT SIGKILL SIGTERM EXIT; '.format(tempdir) +
                'gunzip -c {0}/{1}.txt.gz | distribute-input-lines '.format(source_int_dir, n) +
                ' '.join(['{0}/{1}'.format(tempdir, p) for p in range(num_proc)]) + '& ' +
-               'sort -m {0}'.format(sort_mem_opt) +
-               ' '.join([ '<(get-text-counts {4} {0} <{1}/{2} | sort {3})'.format(ngram_order, tempdir, p, sort_mem_opt,
+               'sort -m {0} '.format(mem_opt) +
+               ' '.join([ '<(get-text-counts {4} {0} <{1}/{2} | sort {3})'.format(ngram_order, tempdir, p, mem_opt,
```

Comment:
I think the reason why we were not detecting the failure was that in the processes substitution here, with expressions like `<(get-text-counts a b c | sort)`, there was no way to pass the error out.  I think we should have some kind of .error file in that directory, and modify these commands to something in the spirit of
 `<(get-text-counts a b c | sort || touch $dir/.error)`
which will make it possible to detect these errors.